### PR TITLE
add install instruction

### PR DESCRIPTION
**Pull request by Grawl**

### CODE REVIEW

**Comment by paazmaya** [comment]
lgtm, @mgreter ?

File: README.md [hunk 1]
```diff
@@ -7,6 +7,29 @@ was initially a port of a previous perl implementation. Its primary intention
 was to bring indented sass syntax support to [`libsass`](https://github.com/sass/libsass).
 `sass2scss` is included in `libsass` since [version 2.0](https://github.com/sass/libsass/releases/tag/v2.0).
 
+## Install
+
+First, download and compile the `sass2scss` binary:
```
Comment (paazmaya) [inline README.md]: Might want to add a bit more clarification of what is needed for compiling

File: README.md [hunk 2]
```diff
@@ -7,6 +7,29 @@ was initially a port of a previous perl implementation. Its primary intention
 was to bring indented sass syntax support to [`libsass`](https://github.com/sass/libsass).
 `sass2scss` is included in `libsass` since [version 2.0](https://github.com/sass/libsass/releases/tag/v2.0).
 
+## Install
```
Comment (paazmaya) [inline README.md]: ... in Linux environment

File: README.md (hunk 1 above)
Comment (Grawl) [inline README.md]: Not sure, I am designer and I work on a Mac 😅

**Comment by mgreter** [comment]
I have added some basic install instructions in https://github.com/mgreter/sass2scss/commit/fcaf40b0b53c77e50d763a05b4ab7c44e32879de.
This will hopefully suffice to give the right pointers.

**Pull request closed without merging**